dev/bots/analyze.dart

```diff
@@ -88,63 +88,63 @@ Future<void> run(List<String> arguments) async {
     foundError(<String>['The analyze.dart script must be run with --enable-asserts.']);
   }
 
-  printProgress('No Double.clamp');
-  await verifyNoDoubleClamp(flutterRoot);
+  //printProgress('No Double.clamp');
```

Comment:
Did you mean to leave all this code commented out?

Reply:
Oops, good catch!